tests/test_codecs.py

```diff
@@ -339,6 +339,20 @@ def test_b3_extract(self):
         span_context = codec.extract(carrier)
         assert span_context.flags == 0x01
 
+        # validate present debug header with falsy value
+        carrier = {'X-b3-SpanId': 'a2fb4a1d1a96d312', 'X-B3-flags': '0',
+                   'X-B3-traceId': '463ac35c9f6413ad48485a3953bb6124'}
```

Comment:
interesting...
```python
>>> header = '463ac35c9f6413ad48485a3953bb6124'
>>> int(header, 16)
93351075330931896558786731617803788580L
```
perhaps supporting 128bit IDs (#228) is not going to be particularly difficult.